src/components/views/right_panel/UserInfo.tsx

```diff
@@ -1442,22 +1442,30 @@ const UserInfoHeader: React.FC<{
         Modal.createDialog(ImageView, params, "mx_Dialog_lightbox", null, true);
     }, [member]);
 
+    const dimensions = UIStore.instance.getElementDimensions("RightPanel");
+    const initialSize = Math.min(dimensions.width, 0.3 * UIStore.instance.windowHeight) - 20; // 2 x 10 margin on each side
+    const [avatarSize, setAvatarSize] = useState(initialSize);
+    useEffect(() => {
+        function setAvatarDimensions(type, entry: ResizeObserverEntry) {
+            const size = Math.min(entry.contentRect.width, 0.3 * UIStore.instance.windowHeight) - 20; // 2 x 10 margin on each side
+            setAvatarSize(size);
+        }
+        UIStore.instance.on("RightPanel", setAvatarDimensions);
+        return () => {
+            UIStore.instance.removeListener("RightPanel", setAvatarDimensions);
+        }
+    }, []);
```

Comment:
The avatar size has 2 states

1: small for when performing a verification
2: large but with a maximal size, the font-size does not scale with the growth of the avatar